source/tutorials/multi-boot/dual-boot-linux.rst

```diff
@@ -0,0 +1,277 @@
+.. _dual-boot-linux:
+
+Dual-boot |CL-ATTR| with Another GRUB-based Linux\* Distro
+##########################################################
+
+In this tutorial, we show how to install another grub-based Linux\* distro 
+alongside |CL|.  To do so, we resize the existing Linux root partition to 
+make enough room to install |CL|. Then we configure the |CL| bootloader, 
+:command:`systemd-boot`, which enables you to dual-boot |CL| and an existing
+Linux distro.  Although we use Ubuntu\* 19.04 Desktop as the example here, 
+these instructions also work for other distros such as Mint Linux, Kubuntu\*, Fedora\*, CentOS\*, among others.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisites
+*************
+
+* Ubuntu 19.04 Desktop is already installed. 
+* There is no unallocated space available, but there is free space 
+  on the Ubuntu root partition.
+* Follow the Preliminary steps that follow below. 
+
+.. include:: ../../get-started/bare-metal-install-desktop.rst
+   :start-after: preliminary-steps-install-desktop:
+   :end-before: install-on-target-start:
```

Comment:
@bktan8. Note that GitHub cannot render an `include` written in reST; therefore, you won't see this section. If you like see the previous document, above, and click the `...` dots menu. Then select View, and see section "Preliminary steps."

Reply:
Got it.